src/dotnet-svcutil/lib/src/Shared/MSBuildProj.cs

```diff
@@ -394,12 +398,26 @@ private static string GetItemValue(XElement reference, string itemName, bool thr
             return null;
         }
 
+        public static bool TryGetItemIdentity(XElement itemName, out string itemIdentity)
```

Comment:
Wasn't sure if we also want to use this method when processing "Reference" nodes (line 234).